docs/zh_cn/kv_int8.md

```diff
@@ -25,12 +25,21 @@ python3 -m lmdeploy.serve.turbomind.deploy internlm-chat-7b /path/to/internlm-ch
 
 ### **第二步**
 
-获取量化参数
+通过以下 2 步，获取量化参数
 
 ```bash
+# 计算 minmax
+python3 -m lmdeploy.lite.apis.calibrate \
+  --model $HF_MODEL \
+  --calib_dataset 'c4' \             # 校准数据集，支持 c4, ptb, wikitext2, pileval
+  --calib_samples 128 \              # 校准集的样本数，如果显存不够，可以适当调小
+  --calib_seqlen 2048 \              # 单条的文本长度，如果显存不够，可以适当调小
+  --work_dir $WORK_DIR \             # 保存 Pytorch 格式量化统计参数和量化后权重的文件夹
+
+# 通过 minmax 获取量化参数
 python3 -m lmdeploy.lite.apis.kv_qparams \
-  --work_dir /path/to/internlm-chat-7b  \             # huggingface 模型目录
-  --turbomind_dir workspace/trition_models/weights/ \ # 保存量化参数的目录
+  --work_dir $WORK_DIR  \                             # 上一步的结果
+  --turbomind_dir workspace/trition_models/weights/ \ # 保存量化参数的目录，推理要用
```

Comment:
Here is a typo: `trition_models` should be `triton_models`.

Reply:
done.